src/kaltura-player.js

```diff
@@ -745,6 +745,10 @@ class KalturaPlayer extends FakeEventTarget {
   get crossOrigin(): ?string {
     return this._localPlayer.crossOrigin;
   }
+
+  get orientationTypes(): PKOrientationType {
```

Comment:
If it's an enum the convention is PascalCase and single body i.e.`OrientationType`